requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
I followed the steps laid out in the docs [here](https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#adding-new-python-requirements) to manually update this dependency, but the content of the generated change in this file worries me a little bit... @wlach does this look ok to you?

Reply:
Yeah that's normal given differences in the python environment. I usually add these types of changes with `git add -p` to *just* add the changes that upgrade the dependency.

If you want to fix this without filing a whole new PR I would run something like `git revert HEAD^` in the branch, make the changes again (or copy them manually from the PR), and then make a new commit.

Reply:
Thanks Will for the detailed instruction, as always!